src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/SortableColumnHeader.cs

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+using System.ComponentModel;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Documents;
+
+namespace NuGet.PackageManagement.UI
+{
+    public class SortableColumnHeader
+    {
+        public static string GetSortPropertyName(DependencyObject obj)
+        {
+            return (string)obj.GetValue(SortPropertyNameProperty);
+        }
+
+        public static void SetSortPropertyName(DependencyObject obj, string value)
+        {
+            obj.SetValue(SortPropertyNameProperty, value);
+        }
+
+        public static ListSortDirection? GetSortDirectionProperty(DependencyObject obj)
```

Comment:
Handle null internally. I'd suggest adding a parameter for a default value to return when the value is null.